src/Greenshot/Helpers/PluginHelper.cs

```diff
@@ -224,10 +224,23 @@ public void LoadPlugins()
                     var pluginEntryName = $"{assemblyName}.{assemblyName.Replace("Greenshot.Plugin.", string.Empty)}Plugin";
                     var pluginEntryType = assembly.GetType(pluginEntryName, false, true);
 
-                    if (CoreConfig.ExcludePlugins != null && CoreConfig.ExcludePlugins.Contains(pluginEntryName))
+                    // the sub namespace from plugin is used to include/exclude plugins

```

Comment:
As suggested in the comments, I'd prefer to keep the configuration backwards compatible *for now* (I think we'll need a clean-up for the plugins in the near future anyway, so we could rename then and officially announce the breaking change in the release notes.)

Idea: maybe we should add a custom metadata attribute (e.g. `ConfigIdentifier` or similar) to each plugin and compare against that? This would also bring the advantage of decoupling the allowlist/blocklist functionality from technical identifiers, which could help prevent it from breaking again in the future. 🤔  What do you think?